test/user.js

```diff
@@ -157,5 +158,55 @@ describe('UserService',function(){
 			_save.restore();
 			done();
 		});
-	})
-});
+	});
+
+	describe('deleteUser', function() {
+		var toDeleteUser;
+		var invalidToDeleteUser;
+		var _destroy;
+		before(function(done) {
+			toDeleteUser = User.forge({ id: 1, email: 'to@delete.com'});
+			invalidToDeleteUser = User.forge({ id: 2, email: 'cant@delete.com'});
+			invalidToDeleteUser.setPassword('password123')
```

Comment:
Does setting a password for this user need to be done? I feel like this is unnecessary.